tests/modules/nf-core/metaeuk/easypredict/test.yml

```diff
@@ -13,3 +13,27 @@
     - path: output/metaeuk/test.headersMap.tsv
       md5sum: b3628c76d7385d326bd8a7931d974cff
     - path: output/metaeuk/versions.yml
+
+- name: metaeuk easypredict test_metaeuk_easypredict_mmseqs
+  command: nextflow run ./tests/modules/nf-core/metaeuk/easypredict -entry test_metaeuk_easypredict_mmseqs -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/metaeuk/easypredict/nextflow.config
+  tags:
+    - metaeuk/easypredict
+    - metaeuk
+  files:
+    - path: output/metaeuk/test.codon.fas
+    - path: output/metaeuk/test.fas
+    - path: output/metaeuk/test.gff
+    - path: output/metaeuk/test.headersMap.tsv
```

Comment:
Any header strings for validation possible for any of these?

Reply:
Unfortunately not - none of these have any header rows, and the fasta headers contain information about the predictions from MetaEuk...